### PR TITLE
Tackle- 913 Memory testing

### PR DESCRIPTION
**Pull request by CarmiWisemon**
MTA Version 6.0

Resolves - https://issues.redhat.com/browse/TACKLE-913

Preview - https://deploy-preview-641--windup-documentation.netlify.app/docs/web-console-guide/master/index.html#memory-testing-requirements-migration-toolkit-for-applications-user-interface

### CODE REVIEW

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc [hunk 1]
```diff
@@ -75,3 +75,39 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
 ** *User name*: admin
 ** *Password*: Passw0rd!
 . When prompted, create a new password.
+
+== Memory testing requirements for {ProductName}
```
Comment (rromannissen) [inline docs/topics/mta-6-installing-web-console-on-openshift.adoc:79]: The section lacks context. These requirements only apply when installing the tool in OpenShift Local, and the way this reads, even on the section title, it feels like this applies on all scenarios. It needs to be more specific.

**Review by rromannissen — CHANGES_REQUESTED** [review]
Please provide more context on the OpenShift Local memory requirement section as detailed on the previous comment.

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc (hunk 1 above)
Comment (anarnold97) [reply docs/topics/mta-6-installing-web-console-on-openshift.adoc:79]: I suggest we change to **Memory testing requirements for installing on OpenShift Local**

Does this work? @rromannissen

Comment (rromannissen) [reply docs/topics/mta-6-installing-web-console-on-openshift.adoc:79]: @anarnold97 sounds great to me!

Comment (anarnold97) [reply docs/topics/mta-6-installing-web-console-on-openshift.adoc:79]: @rromannissen @CarmiWisemon - had another look at this section (Starting line 81 to 106).

Should we be more explicit about the tests to highlight it is MTA Binary Analysis on OpenShift local? 

Trying to focus on the context. Possibly (please, can we have a comma after "The more memory" as it improves readability to me - but it is me being pedantic? ): 

**OpenShift local requires a minimum amount of memory to complete the MTA analysis. The more memory, the quicker the analysis is completed. The following table describes the various amounts of memory required by OpenShift local to complete the MTA binary analysis:

|TABLE|
| ***** |
| Details |

The analysis tests were completed by running the MTA binary analysis through the MTA UI.

The minimum amount of memory required to run MTA binary analysis using OpenShift local is 12 GB.**

Comment (CarmiWisemon) [reply docs/topics/mta-6-installing-web-console-on-openshift.adoc]: I updated the topic based on @anarnold97 @rromannissen  comments. Is this OK?

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc [hunk 2]
```diff
@@ -75,3 +75,42 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
 ** *User name*: admin
 ** *Password*: Passw0rd!
 . When prompted, create a new password.
+
+== Memory testing requirements for installing on OpenShift Local
+
+OpenShift local requires a minimum amount of memory to complete the {ProductShortName} analysis. The more memory, the quicker the analysis is completed. The table below describes the various amounts of memory required by OpenShift local to complete the MTA binary analysis:
```
Comment (rromannissen) [inline docs/topics/mta-6-installing-web-console-on-openshift.adoc:81]: Please use the correct style for the product name: "OpenShift Local". I would also add a hyperlink to the product page in at least one of the mentions: https://developers.redhat.com/products/openshift-local/overview

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc [hunk 3]
```diff
@@ -75,3 +75,42 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
...
+|MTA works and the analysis is completed in less than 2 minutes
+
+|`20`
+|MTA works quickly and the analysis is completed in less than 1 minute
+|====
+
+
+The analysis tests were completed by running the MTA binary analysis through the MTA UI.
+
+The minimum amount of memory required to run MTA in OpenShift local is *12 GB*.
```
Comment (rromannissen) [inline docs/topics/mta-6-installing-web-console-on-openshift.adoc:109]: Please use "OpenShift Local", not "OpenShift local"

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc [hunk 4]
```diff
@@ -75,3 +75,42 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
...
+
+[NOTE]
+====
+** All the analyses were performed using the *tackle-testapp* binary.
+** All the tests were conducted on a local OpenShift cluster, without the monitoring tools installed. Installing the cluster monitoring tools requires an additional 5 GB of memory.
```
Comment (rromannissen) [inline docs/topics/mta-6-installing-web-console-on-openshift.adoc:114]: Should read "All the tests were conducted on an OpenShift Local cluster"

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc [hunk 5]
```diff
@@ -75,3 +75,42 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
...
+
+|`15`
+|MTA works and the analysis is completed in less than 2 minutes
+
+|`20`
+|MTA works quickly and the analysis is completed in less than 1 minute
+|====
+
+
+The analysis tests were completed by running the MTA binary analysis through the MTA UI.
```
Comment (rromannissen) [inline docs/topics/mta-6-installing-web-console-on-openshift.adoc:107]: MTA User Interface to keep consistency with the rest of the document

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc [hunk 6]
```diff
@@ -75,3 +75,42 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
 ** *User name*: admin
 ** *Password*: Passw0rd!
 . When prompted, create a new password.
+
+== Memory testing requirements for installing on Red Hat OpenShift Local
```
Comment (sbeskin-redhat) [inline docs/topics/mta-6-installing-web-console-on-openshift.adoc:79]: @CarmiWisemon  
Memory requirements for running {ProductShortName} on Red Hat OpenShift Local

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc [hunk 7]
```diff
@@ -75,3 +75,42 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
...
+
+https://developers.redhat.com/products/openshift-local/overview[Red Hat OpenShift Local] requires a minimum amount of memory to complete the {ProductShortName} analysis. The more memory, the quicker the analysis is completed. The table below describes the various amounts of memory required by OpenShift Local to complete the MTA binary analysis:
```
Comment (sbeskin-redhat) [inline docs/topics/mta-6-installing-web-console-on-openshift.adoc:81]: @CarmiWisemon 
When installed on https://developers.redhat.com/products/openshift-local/overview[Red Hat OpenShift Local], {ProductShortName} requires a minimum amount of memory to be able to complete its analysis. Adding memory above the required minimum makes the analysis process run faster. The table below describes the MTA performance with varying amounts of memory.

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc [hunk 8]
```diff
@@ -75,3 +75,42 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
...
+
+.OpenShift Local memory testing requirements
+[cols="25%,75%", options="header"]
+|====
+|Memory (GB)
+|Description
+
+
+|`10`
+|MTA cannot run the analysis due to insufficient memory
```
Comment (sbeskin-redhat) [inline docs/topics/mta-6-installing-web-console-on-openshift.adoc:91]: @CarmiWisemon 
"{ProductShortName} cannot run the analysis due to insufficient memory."

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc [hunk 9]
```diff
@@ -75,3 +75,42 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
...
+
+|`11`
+|MTA cannot run the analysis due to insufficient memory
```
Comment (sbeskin-redhat) [inline docs/topics/mta-6-installing-web-console-on-openshift.adoc:94]: @CarmiWisemon 
"{ProductShortName} cannot run the analysis due to insufficient memory."

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc [hunk 10]
```diff
@@ -75,3 +75,42 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
...
+
+|`12`
+|*MTA works and the analysis is completed in approximately 3 minutes*
```
Comment (sbeskin-redhat) [inline docs/topics/mta-6-installing-web-console-on-openshift.adoc:97]: @CarmiWisemon 
"{ProductShortName} completes the analysis in approximately 3 minutes."

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc [hunk 11]
```diff
@@ -75,3 +75,42 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
...
+
+|`15`
+|MTA works and the analysis is completed in less than 2 minutes
```
Comment (sbeskin-redhat) [inline docs/topics/mta-6-installing-web-console-on-openshift.adoc:100]: @CarmiWisemon 
"{ProductShortName} completes the analysis in less than 2 minutes."

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc [hunk 12]
```diff
@@ -75,3 +75,42 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
...
+
+|`20`
+|MTA works quickly and the analysis is completed in less than 1 minute
```
Comment (sbeskin-redhat) [inline docs/topics/mta-6-installing-web-console-on-openshift.adoc:103]: @CarmiWisemon 
"{ProductShortName} completes the analysis in less than 1 minute."

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc [hunk 13]
```diff
@@ -75,3 +75,42 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
...
+|====
+
+
+The analysis tests were completed by running the MTA binary analysis through the MTA User Interface.
```
Comment (sbeskin-redhat) [inline docs/topics/mta-6-installing-web-console-on-openshift.adoc:107]: @CarmiWisemon 
"We performed the tests by running the {ProductShortName} binary analysis through the {WebName}."
Move to Notes below.

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc [hunk 14]
```diff
@@ -75,3 +75,42 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
...
+
+The minimum amount of memory required to run MTA in OpenShift Local is *12 GB*.
```
Comment (sbeskin-redhat) [inline docs/topics/mta-6-installing-web-console-on-openshift.adoc:109]: @CarmiWisemon 
"The test results indicate that the minimum amount of memory for running {ProductShortName} on OpenShift Local is *12 GB*."

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc [hunk 15]
```diff
@@ -75,3 +75,42 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
...
+
+[NOTE]
```
Comment (sbeskin-redhat) [inline docs/topics/mta-6-installing-web-console-on-openshift.adoc:111]: @CarmiWisemon 
NOTES

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc [hunk 16]
```diff
@@ -75,3 +75,42 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
...
+====
+** All the analyses were performed using the *tackle-testapp* binary.
```
Comment (sbeskin-redhat) [inline docs/topics/mta-6-installing-web-console-on-openshift.adoc:113]: @CarmiWisemon 
** We performed the tests by running the {ProductShortName} binary analysis through the {WebName}.
** We performed all the analyses using the *tackle-testapp* binary.

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc [hunk 17]
```diff
@@ -75,3 +75,42 @@ You can install the {ProductName} ({ProductShortName}) and the {WebName} on Open
...
+** All the tests were conducted on an OpenShift Local cluster, without the monitoring tools installed. Installing the cluster monitoring tools requires an additional 5 GB of memory.
```
Comment (sbeskin-redhat) [inline docs/topics/mta-6-installing-web-console-on-openshift.adoc:114]: @CarmiWisemon 
** We conducted all the tests on an OpenShift Local cluster without the monitoring tools installed. Installing the cluster monitoring tools requires an additional 5 GB of memory.

File: docs/topics/mta-6-installing-web-console-on-openshift.adoc (hunk 13 above)
Comment (CarmiWisemon) [reply docs/topics/mta-6-installing-web-console-on-openshift.adoc:107]: I'm leaving this one as is.

**Pull request merged**